test/e2e/test_deb.py

```diff
@@ -39,12 +41,24 @@ def deb_archive_with_cartridge(cartridge_cmd, tmpdir, project_with_cartridge, re
                 "stress\n" +
                 "neofetch < 25")
 
+    net_msg_max = 1024
+
+    systemd_unit_params = os.path.join(tmpdir, "systemd-unit-params.yml")
+    with open(systemd_unit_params, "w") as f:
+        f.write(f"""
+                instance-env:
+                    net-msg-max: {net_msg_max}
+                """)
+
+    patch_init_to_net_msg_max(project, net_msg_max)
```

Comment:
I suggest testing not only `net_msg_max`, but also custom params like `some-user-param`. And params for stateboard.